django_loose_fk/fields.py

```diff
@@ -106,15 +108,15 @@ def _add_check_constraint(
             options.original_attrs["constraints"] = options.constraints
         return
 
-    @property
+    @cached_property
```

Comment:
@sergei-maertens made some more properties cached, it doesn't make a huge difference, but since the return values of these properties doesn't change after the instance is initialized, making it cached avoids it being computed for every result (e.g for a list of 100 zaken, this was ran once for each zaak.zaaktype)

Reply:
I'm not looking at this too much since I'm not using this library myself anymore - I think it's probably okay so that's why I'm approving, but do validate within team-bron :)